### PR TITLE
MEED-288: Edit note item icon in favorite drawer

### PR DESCRIPTION
**Pull request by SaraBoutej**
This PR allow to edit the note icon in the favorite drawer by a new image according to the design.

### CODE REVIEW

**Review by boubaker — APPROVED** [review]

File: notes-webapp/src/main/webapp/vue-app/notes-extensions/components/favorites/NotesFavoriteItem.vue [hunk 1]
```diff
@@ -1,7 +1,10 @@
 <template>
   <v-list-item class="clickable" :href="noteUrl">
     <v-list-item-icon class="me-3 my-auto">
-      <v-icon size="22" class="icon-default-color"> fas fa-clipboard </v-icon>
+      <v-img
+        :src="noteImg"
+        max-height= "28"
+        max-width= "25" />
```
Comment (boubaker) [inline notes-webapp/src/main/webapp/vue-app/notes-extensions/components/favorites/NotesFavoriteItem.vue:7]: Can you reduce the image size please in Bytes knowing that its height and width will be very small comparing to its resolution. @Samuel-Renault-eXo ( To check with Sofien please)

Comment (srenault-meeds) [reply notes-webapp/src/main/webapp/vue-app/notes-extensions/components/favorites/NotesFavoriteItem.vue:7]: I propose these two files. Up to you decide which one to take
notes-appicon-review
notes-appicon-reviewSmaller
notes-appicon-reviewSmaller1
.

Comment (srenault-meeds) [reply notes-webapp/src/main/webapp/vue-app/notes-extensions/components/favorites/NotesFavoriteItem.vue:7]: Last proposed and validated image here
notes-appicon

Comment (boubaker) [reply notes-webapp/src/main/webapp/vue-app/notes-extensions/components/favorites/NotesFavoriteItem.vue:7]: ok, thanks

**Pull request merged**